src/InterpreterGenerator.h

```diff
@@ -628,15 +629,15 @@ class NodeGenerator : public RamVisitor<std::unique_ptr<InterpreterNode>> {
      */
     inline std::vector<const RamCondition*> toConjunctionList(const RamCondition* condition) {
         std::vector<const RamCondition*> list;
-        std::stack<const RamCondition*> stack;
+        std::queue<const RamCondition*> queue;
```

Comment:
Could you rename `queue` to be more informative?

`toCheck` or `conditionsToCheck`, perhaps?

`list` could be `conjunctionList`